docs/canvas/canvas-present-workpad.asciidoc

```diff
@@ -0,0 +1,30 @@
+[role="xpack"]
+[[canvas-present-workpad]]
+== Present your workpad
+
+When you are ready to present your workpad, use and enable the presentation options.
+
+[float]
+[[view-fullscreen-mode]]
+=== View your workpad in fullscreen mode
+
+In the upper left corner, click the *Enter fullscreen mode* icon.
+
+[float]
+[[zoom-in-out]]
+=== Use the zoom options
+
+In the upper left corner, click the *Zoom controls* icon, then select one of the options.
+
+[float]
+[[enable-autoplay]]
+=== Enable autoplay 
```

Comment:
Should we reiterate the refresh controls stuff here as well since it's in the screenshot? I think it's possible for a user to stumble upon this page without reading the elements page.

Reply:
I don't want to double document this feature, especially since it doesn't really relate to `Present your workpad`.